services/concept_insights/ci.js

```diff
@@ -333,9 +332,10 @@ module.exports = function (RED) {
             var message = "Creating document error: Missing document name, label or content type";
             return node.error(message, msg);
           }
-        }          
+        }    
 
         function checkStatus(fileLength) {
+
           //Calculate interval based on length of file, default to 10 seconds
           var interval = 10000;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'interval'.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2394968359/issues/source?bid=3225115&fileBranchId=3230627#l341)